spec/system/assignment/edit_spec.rb

```diff
@@ -68,9 +68,10 @@
     it 'stops them from changing assignments they do not own' do
       select(new_user.last_name, from: 'User')
       click_button 'Save'
-      within('div.alert.alert-danger') do
-        expect(page).to have_selector 'li', text: 'You may only edit'
-      end
+      expect(page).to have_selector '.alert.alert-danger',
+                                    text: 'You may only edit or create assignments such that you become on call. ' \
```

Comment:
Are you able to use the localized format here? might as well, right? Unless i suppose you reaalllyyy want it to be very explicit

Reply:
This isn't in a controller so we don't have access to the locales.